app/experimenter/experiments/constants.py

```diff
@@ -692,6 +694,27 @@ class ExperimentConstants(object):
     QA_STATUS_HELP_TEXT = """
     """
 
+    # Results
+    RESULTS_URL_HELP_TEXT = """
+        This is the link to final results.
+    """
+    RESULTS_INITIAL_HELP_TEXT = """
+        This is the place for links to any early findings or preliminary
+        results, while the final results are being developed.  If there
+        are not results for this specific experiment, because they are
+        associated with a related experiment - this is the place to leave
+        a trail to find those closely experiment and/or results.
+    """
+
+    RESULTS_LESSONS_HELP_TEXT = """
+        What went well or did not go well with any part of the
```

Comment:
Oh I didn't realize we were putting a section for feedback about this, that's awesome, this is exactly what I want!